target/linux/realtek/image/rtl930x.mk

```diff
@@ -63,3 +63,34 @@ define Device/zyxel_xgs1250-12
 	uImage gzip
 endef
 TARGET_DEVICES += zyxel_xgs1250-12
+
+define Device/plasmacloud-common
+  SOC := rtl9302
+  UIMAGE_MAGIC := 0x93000000
+  DEVICE_VENDOR := Plasma Cloud
+  IMAGE_SIZE := 15872k
```

Comment:
`check-size` missing:

```
IMAGE/factory.bin := append-kernel | append-rootfs | pad-rootfs | check-size
```

I am not sure if there was ever anything added for `sysupgrade-tar`. Because it is necessary to check here for both kernel + rootfs and not only the rootfs (when you add it after `pad-rootfs`) or the whole tar file (when you add it after `sysupgrade-tar`).